sample/src/main/java/io/getstream/chat/android/client/sample/DocumentationSamplesJava.java

```diff
@@ -108,7 +108,9 @@ public void onSuccess(@NotNull ConnectionData data) {
 
                     @Override
                     public void onError(@NotNull ChatError error) {
-                        error.printStackTrace();
+                       if (error.getThrowable() != null) {
+                           error.getThrowable().printStackTrace();
```

Comment:
In some examples of this file, we save the `Throwable` into a local variable before null checking it, and sometimes we fetch it twice, like here. I prefer the other approach, as it's more meaningful and logically correct.

Here it could happen (though it won't in practice) that `getThrowable()` returns different values the two times we call it, and therefore the `null` check is invalid.

Reply:
It's because linter. Linter sees the val as final property, and such twice re-calculation for linter is like invocation value from final field. That's why here it works.
But when you have Result<T> it has also final property error, which has also final property throwable. Here linter doesn't let to do. That's why it's better to copy in local variable.
I always try to avoid Java's verbosity that's why here two different styles.